### DISCUSSION TITLE
Half working cart reader

### DISCUSSION THREAD

**Discussion by CuccoEgg** [sanni/cartreader#175] [Need help]
I just made two cart readers, one worked perfectly fine and the other has weird problems. It wont properly read cart titles, wont write to SD card, and randomly say SD card error after any read or write function to the SD card. Menu is working fine and can cycle through things. Does anyone know what could be wrong? 

I tried swapping the Arduino and the sd card module, tried 3 different chargers (all at least 5V/2A) and still have the problem

**Comment by sanni** [comment]
Ideas:
- Make sure there is no short circuit somewhere on the PCB by using a multi-meter to test continuity between neighboring pins.
- Check if the GBA slot has bridged pins, not only the solder pads but also the vertical pins running along the backside of the slot. 
- Inspect the PCB for flaked off solder mask. 
- Re-solder all the switches, they tend to produce cold solder joints because of their high thermal mass
- Make sure you de-soldered the 2 components from the Arduino.
